### ISSUE TITLE
Colored circles in Glance cards based on entity active state

### ISSUE THREAD

**Issue by zappasmart** [home-assistant/home-assistant-polymer#5040]
## State of the art
Glance cards currently provide a way to color the entity **icon** based on its active state:

![image](https://user-images.githubusercontent.com/53600373/75666413-0f407400-5c76-11ea-9d1a-213144b61bce.png)

## The request
It would be nice if there was the option of coloring the **circle** behind the icon, instead of the icon, so the active state stands out better, either when it's on or off:

![image](https://user-images.githubusercontent.com/53600373/75665020-c25b9e00-5c73-11ea-9320-16a90f8f91f9.png)

![image](https://user-images.githubusercontent.com/53600373/75664819-6264f780-5c73-11ea-9eb1-8ead20a82521.png)

Readability improves.

## Additional information
No change to the HTML structure is needed, as it would be just a matter of styling the already present `<state-badge>` tag, that represents the circle behind the icon:

![image](https://user-images.githubusercontent.com/53600373/75667080-49f6dc00-5c77-11ea-8d26-f6fa5299d04b.png)

**Comment by zsarnett** [comment]
This could be inconsistent with other cards. Ex: Button Card

**Comment by zappasmart** [comment]
You are correct. The concept here is colouring the background of an icon, instead of colouring the icon, based on the active state, for a faster status recognition. Entity card and Button card should be considered as well.

**Comment by iantrich** [comment]
I do like this idea for Glance and only Glance

**Comment by matthiasdebaat** [comment]
I like this! We should do this for the Glance card
